torch/_refs/__init__.py

```diff
@@ -3291,7 +3291,9 @@ def reshape_as(self: TensorLikeType, other: TensorLikeType) -> TensorLikeType:
 
 @register_decomposition(aten.roll)
 def roll(
-    a: TensorLikeType, shifts: DimsType, dims: DimsType = tuple()
+    a: TensorLikeType,
+    shifts: Sequence[Union[int, torch.types.SymInt]],
```

Comment:
Will need to change this since integers shifts are supported by `torch.roll`.

(There is a comment in the func body below about ints being auto-expanded.)